python_modules/dagster/dagster/core/asset_defs/source_asset.py

```diff
@@ -24,8 +24,7 @@ class SourceAsset(
         ],
     )
 ):
-    """A SourceAsset represents an asset that is not generated by any Dagster op in the repository
-    that it's referenced from.
+    """A SourceAsset represents an asset that will be loaded by, but not updated by, a given job.
```

Comment:
I think source assets are a valid concept even in a world without jobs. Is there a way to document this in a way that communicates that?

Reply:
maybe i'll just make it more vague and say "loaded by (but not updated by) Dagster".

Reply:
sounds good